src/services/ActivityService.ts

```diff
@@ -4,8 +4,9 @@ import { toUint8Array } from '@smithy/util-utf8';
 import { LambdaService } from './LambdaService';
 import { Configuration } from '../utils/Configuration';
 import { ACTIVITY_TYPE, ERRORS } from '../utils/Enums';
-import { IActivity, IActivityParams, IInvokeConfig } from '../models';
+import { IActivityParams, IInvokeConfig } from '../models';
 import { validateInvocationResponse } from '../utils/validateInvocationResponse';
+import { ActivitySchema } from "@dvsa/cvs-type-definitions/types/v1/activity";
 import HTTPError from '../models/HTTPError';
 
 class ActivityService {
```

Comment:
Can the docblocks have the return types added to them.
Are you able to add a type to endVisit instead of using the any?

Reply:
We don't appear to add return types to the docblocks in our other repos, eg atf / retro that was changed recently.

I did look at specifying the return type, but unlike other methods we have updated as part of this process, it's not returning a Promise<any> in lieu of a Promise<TestResultSchema> or Promise<ActivitySchema>.

To implement would probably require a new custom response object, to handle the variety of potential promises that can be returned, and a rework of the logic that is currently there. 

Can take a look but would change the scope of the ticket and would be going into the realms of a refactor then?